notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -10571,6 +10571,46 @@ father and son [we saw brief flashbacks to Truman with his dad who went missing
 mother and son [we saw Truman with his mom from time to time]
 
 
+movie: The XFiles Fight the Future (1998)
+=========================================
+
+:: Title
+The X-Files: Fight the Future
+
+:: Description
+The X-Files (also known as The X-Files: Fight the Future) is a 1998 American
+science fiction thriller film based on Chris Carter's television series of
+the same name, which revolves around fictional unsolved cases called the
+X-Files and the characters solving them.
+
+:: Date
+1998-06-19
+
+:: References
+https://en.wikipedia.org/wiki/The_X-Files_(film)
+
+:: Ratings
+3 <paul>
+
+:: Genre
+sci-fi
+
+:: Major Themes
+secret society conspiracy theory [The story follows the FBI agents Mulder and Skully as they attempt to get to the bottom of a global conspiracy being perpetrated by an unnamed shadowy organization to prepare the way for Earth to be colonized by space aliens.]
+subversive life form from outer space [A shadowy organization with a global presence was preparing the way for space aliens to colonize Earth.]
+extraterrestrial being [Powerful space aliens were using a shadowy organization to spread a virus around the world as a prelude to colonization.]
+human characters in contrast [The intuition-trusting, paranormal-believing Agent Mulder was pointedly juxtaposed with his rational-minded, skeptical partner Agent Skully.]
+human interpersonal relationship [Much focus was placed on the relationship between the buddy FBI agents Mulder and Skully. In one subplot, Mulder was urging Skully to stay on the force when she wanted to quit the FBI and return to practicing medicine.]
```

Comment:
Not for the first time do I think we should have a theme for coworker and coworker, or maybe working partners

Reply:
I see this is the third case of two law enforcement partners <https://themeontology.org/theme.php?name=human%20interpersonal%20relationship> (a.k.a. "buddy cops"). And it's anyone's guess as to how many other buddy cop stories I missed putting this for. I can think of each of the six Alien Nation films <https://themeontology.org/story.php?name=Collection%3A%20Alien%20Nation%20Films> off the top of my head.

Reply:
Well why don't we just add a "buddy cop" theme then!

Reply:
I like it! I'll handle it as a separate merge that will include all the instances that I can think of along with an offering for the defined theme.